airflow/models/dagrun.py

```diff
@@ -1661,7 +1661,5 @@ def __init__(self, content, user_id=None):
         self.user_id = user_id
 
     def __repr__(self):
-        prefix = f"<{self.__class__.__name__}: {self.dag_id}.{self.dagrun_id} {self.run_id}"
-        if self.map_index != -1:
-            prefix += f" map_index={self.map_index}"
+        prefix = f"<{self.__class__.__name__}: {self.dag_run.dag_id}.{self.dag_run_id} {self.dag_run.run_id} {self.content}"
```

Comment:
This should still show map_index. Honestly I’m not event sure why `dag_run_id` should be shown, it is just a number and not really useful in any way.